frontend/packages/dev-console/src/components/pipelineruns/__tests__/PipelineRunDetailsPage.spec.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { shallow } from 'enzyme';
+import { referenceForModel } from '@console/internal/module/k8s';
+import { DetailsPageProps } from '@console/internal/components/factory';
+import { PipelineRunModel } from '../../../models';
+import PipelineRunDetailsPage from '../PipelineRunDetailsPage';
+
+const pipelineRunDetailsProps: DetailsPageProps = {
```

Comment:
can use:
```suggestion
const pipelineRunDetailsProps: React.ComponentProps<typeof PipelineRunDetailsPage> = {
```
and remove the `DetailsPageProps` import

Reply:
done